install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     version: master
 rules:
 - apiGroups: ["","apps", "autoscaling"]
+  attributeRestrictions: null
```

Comment:
Why add this back?

Reply:
Yes, please remove them as they were causing problems: #6584

Reply:
@gbaufake when you remove them here, please write up another PR to remove them from the yaml in the kiali github repo as well).